layouts/community/ortho_4x12/symbolic/keymap.c

```diff
@@ -19,6 +19,12 @@ enum custom_keycodes {
   R_RAISE
 };
 
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```